subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/mqtt/MQTTNotificationService.kt

```diff
@@ -0,0 +1,68 @@
+package com.egm.stellio.subscription.service.mqtt
+
+import com.egm.stellio.shared.model.BadSchemeException
+import com.egm.stellio.subscription.model.Notification
+import com.egm.stellio.subscription.model.Subscription
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.eclipse.paho.client.mqttv3.MqttException as MqttExceptionV3
+import org.eclipse.paho.mqttv5.common.MqttException as MqttExceptionV5
+
+@Service
+class MQTTNotificationService(
+    @Value("\${mqtt.clientId}")
+    private val clientId: String = "stellio-context-brokerUrl",
+    private val mqttVersionService: MQTTVersionService
+) {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    suspend fun mqttNotifier(
+        subscription: Subscription,
+        notification: Notification,
+        headers: Map<String, String>
+    ): Boolean {
+        val endpoint = subscription.notification.endpoint
+        val uri = endpoint.uri
+        val userInfo = uri.userInfo.split(':')
+        val username = userInfo.getOrNull(0) ?: ""
+        val password = userInfo.getOrNull(1)
+        val brokerScheme = Mqtt.SCHEME.brokerSchemeMap[uri.scheme]
+            ?: throw BadSchemeException("${uri.scheme} is not a valid mqtt scheme")
```

Comment:
better check this at subscription creation / modification time. And return a 400 if scheme is invalid.